numpy/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+# very simple, just enough to start running tests
+
```

Comment:
We should add annotations `def __getattr__(name) -> Any: ...` to the module and `def __getattr__(self, name) -> Any: ...` to `ndarray`/`dtype` to mark them as flexibly typed.

Reply:
Not sure what you mean about adding it to the module. Is that supported in type stubs? It's certain not supported in normal python,

Reply:
Well, OK, we can leave it out for now. It is indeed valid in stub files (only).